test/Orchard.Tests/DisplayManagement/Decriptors/DefaultShapeTableManagerTests.cs

```diff
@@ -126,7 +125,7 @@ public TestThemeExtensionInfo(string name, IFeatureInfo baseTheme)
                     {"name", name},
                     {"desciption", name},
                     {"type", "theme"},
-                    {"basetheme", baseTheme.Id}
+                    {"basethemes", baseTheme.Id }
```

Comment:
Is it to represent the list of base themes in the whole hierarchy? or do you really mean a theme can inherit from two independent ones?